packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -131,41 +159,33 @@ function FolderImpl({ item, anchors }: FolderProps) {
   )
 }
 
-interface SeparatorProps {
-  title?: string
-  topLevel: boolean
-}
-
-function Separator({ title, topLevel }: SeparatorProps): ReactElement {
-  // since title can be empty string ''
-  const hasTitle = title !== undefined
```

Comment:
title can't be empty string, will fallback on frontMatter.tittle or file page name